src/styles/_SearchBar.scss

```diff
@@ -0,0 +1,31 @@
+.SearchBar {
```

Comment:
The search bar and filters look great! One suggestions, it might be easier to optimize them for desktop first in terms of styling and then add media queries to stack them for smaller screens.

Reply:
Yes good point! We can make those changes this afternoon 😀